StatusKeycard module not compatible with Xcode 14.0 / iOS 16.0

# Bug Report

## Problem
StatusKeycard module not compatible with Xcode 14.0 / iOS 16.0


#### Expected behavior
`make run-ios` should build without failure.

#### Actual behavior
`make run-ios` results with the following error:

```bash
** BUILD FAILED **


The following build commands failed:
	SwiftEmitModule normal x86_64 Emitting\ module\ for\ react_native_status_keycard (in target 'react-native-status-keycard' from project 'Pods')
(1 failure)
```

The error is in the `react-native-status-keycard` package [here](https://github.com/status-im/react-native-status-keycard/blob/af61b0213adbf4b5a6bae97c85496d95007d5acb/ios/StatusKeycard.swift#L12)

and looks like so:

```swift
Stored properties cannot be marked potentially unavailable with '@available'
```

### Reproduction

1) Run `make run-ios` on a MacBook with Xcode 14.0 installed

### Additional Information

- Status version: nightly 
- Operating System: iOS
- Development machine OS: macOS

--- a/package.json
+++ b/package.json
@@ -68,7 +68,7 @@
     "react-native-safe-area-context": "^2.0.0",
     "react-native-shake": "^3.3.1",
     "react-native-share": "^7.0.1",
-    "react-native-status-keycard": "git+https://github.com/status-im/react-native-status-keycard.git#refs/tags/v2.5.37",
+    "react-native-status-keycard": "git+https://github.com/status-im/react-native-status-keycard.git#refs/tags/v2.5.38",
     "react-native-svg": "^9.8.4",
     "react-native-touch-id": "^4.4.1",
     "react-native-webview": "git+https://github.com/status-im/react-native-webview.git#refs/tags/v11.16.0-status",

